laboite/apps/cryptocurrency/models.py

```diff
@@ -24,12 +26,18 @@ class AppCryptocurrency(App):
         ('USD', _('Dollars')),
         ('EUR', _('Euros')),
     )
-    cryptocurrency = models.CharField(_('Crypto-monnaie'), help_text=_('Veuillez indiquer la crypto-monnaie que vous souhaitez convertir'), max_length=16, default='bitcoin', choices=CRYPTOCURRENCY_CHOICES)
-    currency = models.CharField(_('Monnaie'), help_text=_('Veuillez indiquer la devise dans laquelle vous souhaitez convertir'), max_length=8, default='EUR', choices=CURRENCY_CHOICES)
-    value = models.PositiveSmallIntegerField(_('Valeur'), default=0)
+    cryptocurrency = models.CharField(_('Crypto-monnaie'),
+                                      help_text=_('Veuillez indiquer la crypto-monnaie que vous souhaitez convertir'),
+                                      max_length=16, default='bitcoin', choices=CRYPTOCURRENCY_CHOICES)
+    currency = models.CharField(_('Monnaie'),
+                                help_text=_('Veuillez indiquer la devise dans laquelle vous souhaitez convertir'),
+                                max_length=8, default='EUR', choices=CURRENCY_CHOICES)
+    value = models.DecimalField(_('Valeur'), max_digits=8, decimal_places=3, default=0)
 
     def _get_data(self):
-        width = len(str(self.value)) * 5
+        #  Convert as str and keep only 5 chars max (without decimal separator)
+        value = str(self.value)[:5].rstrip('.')
```

Comment:
If we have 5780.558 it will display 5780€ if we have 446.6654 it will display 446.6€ if we have 0.55687879 it will display 0.556€ ?

Reply:
Il y a des tests: https://github.com/laboiteproject/lenuage/pull/89/files#diff-a8c15ad96c45f0b4d5e0d020d830f4e9R225 \o/

Reply:
Exactly, this isn't perfect but since the available space is limited...